projects/core/src/occ/occ-models/occ.models.ts

```diff
@@ -1431,6 +1431,52 @@ export namespace Occ {
      * @member {boolean} [updateable]
      */
     updateable?: boolean;
+    /**
+     * @member {StatusSummary[]} [statusSummaryList]
+     */
+    statusSummaryList?: StatusSummary[];
+    /**
+     * @member {ConfigurationInfo[]} [configurationInfos]
+     */
+    configurationInfos?: ConfigurationInfo[];
+  }
+
+  /**
+   *
+   * An interface representing ConfigurationInfo.
```

Comment:
Please describe in a more detailed way what this is about. Important to mention that this is not bound to a specific configurator but a data that is common for all configurators.

Reply:
I'll do. But all other interfaces in that file (except enum definitions) are documented in the same way.